test/config-next/sa.json

```diff
@@ -4,6 +4,7 @@
     "maxDBConns": 10,
     "maxConcurrentRPCServerRequests": 16,
     "debugAddr": "localhost:8003",
+    "CertStatusOptimizationsMigrated": false,
```

Comment:
I left this `false` in `config-next/sa.json` because it shouldn't be assumed true until verified against the DB.